source/_integrations/sharkiq.markdown

```diff
@@ -21,21 +20,78 @@ The `sharkiq` integration allows you to control your [Shark IQ](https://www.shar
+<div class='note'>
+This integration uses your device's cloud API to communicate with the device.  It has been tested with the R101AE and
+AV2001 robots, but should be compatible with all smart SharkIQ robot vacuums.  If you are having issues with your device not
+responding as expected, please <a href="https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sharkiq%22" target="_blank" rel="noreferrer">open an issue on Github</a>.
+</div>
 
 {% include integrations/config_flow.md %}
 
-## Services
+## Entities
+A new device & entity will be created for each robot vacuum you have linked to your SharkIQ account.
+
+The state of your robot entities will be the current status of the robot (i.e `docked`, `cleaning`, `paused`, `returning`, etc).
 
-Currently supported services are:
+Each robot entity also has a series of attributes to help provide you with more details about your robot's state and specific sensors.
 
+| Attribute              | Type        | Description                                            |
+|------------------------|-------------|--------------------------------------------------------|
+| `fan_speed_list`       | `list`      | A list of fan speeds supported by the device.          |
+| `battery_level`        | `int`       | Battery charge percentage as an `integer` from 0-100.  |
+| `battery_icon`         | `string`    | Icon associated with the `battery_level` attribute.    |
+| `fan_speed`            | `string`    | Fan speed currently set on the device.                 |
+| `last_error_code`      | `int`       | Most recent error code returned by the device.         |
+| `last_error_message`   | `string`    | Most recent error message returned by the device.      |
+| `rssi`                 | `int`       | Wifi Received Signal Strength Indicator (RSSI) value.  |
+| `recharge_and_resume`  | `bool`      | Boolean value of the Recharge and Resume feature flag. |
+| `rooms`                | `list[Str]` | A list of rooms configured for your robot vacuum.      |
+
+## Services
+### Native `vacuum` Services
+The following native `vacuum` services are supported:
 - `start`
 - `pause`
 - `stop`
 - `return_to_base`
 - `locate`
 
+For usage and required fields, refer to the [vacuum component services](https://www.home-assistant.io/integrations/vacuum/#component-services).
+
+<div class='note'>
+Support for <code>pause</code> may vary depending on the model of your vacuum.  If you notice the `pause` service does not seem
+to be working, use the <code>stop</code> service instead.  In this case the robot should pause, and can be resumed by
+calling the <code>start</code> service again.  To completely cancel the clean job, use `return_to_base`.
+</div>
+
+### SharkIQ-Specific Services
+#### `clean_room`
+If your robot vacuum supports cleaning specific rooms, you can call this service to target a room or multiple rooms,
+without starting a full cleaning job.
+
+
+<div class='note'>
```

Comment:
Let's drop these note boxes. This can just be described without those. Also, remove the use of double spaces. Remove HTML as well.